AdoptMeProject/adopt-me/src/components/Layout/Layout.js

```diff
@@ -9,29 +9,20 @@ export const Layout = () => {
   return (
     <Box
       sx={{
-        backgroundImage: 'url(images/backgroundImage.jpg)',
-        backgroundSize: 'cover',
-        backgroundPosition: 'center',
-        backgroundRepeat: 'no-repeat',
+        backgroundImage: "url(images/backgroundImage.jpg)",
+        backgroundSize: "cover",
+        backgroundPosition: "center",
+        backgroundRepeat: "no-repeat",
+        justifyContent: "space-between",
+        display: "flex",
+        minHeight: "100vh",
+        flexDirection: "column",
```

Comment:
There are properties of the sx that could be used as properties of the components itself. When there is a case like that, try to use the default props of the component, so that you could skip defining them in the sx directive. Checkout the examples here: https://mui.com/material-ui/react-box/